docs/src/cookbooks/cookbook.md

```diff
@@ -404,6 +406,39 @@ class ModuleWithOptionalIO(flag: Boolean) extends Module {
 }
 ```
 
+### How do I create a single input/output wire for a bundle?
+
+If you have a bundle with many elements in a module interface, the interface can become
+quite verbose. To simplify the module interface, the entire bundle
+can be represented with a single wire.
+
```

Comment:
This recipe is very similar to "How do I create a UInt from an instance of a Bundle?" and "How do I create a Bundle from a UInt?", but I totally agree that having the right framing of recipes is important for discoverability. 

What do you think about phrasing it as something like "How do I pack an Aggregate into a single input/output port?" and then pointing it to "How do I create a UInt from an instance of a Bundle?"? We could also tweak the example up there to provide a port as well (instead of just a wire).

Reply:
"Aggregate" is not used elsewhere in the cookbook, but rather "Bundle" is.

Tweaked exampel as suggested.